.eslintrc

```diff
@@ -1,10 +1,43 @@
 extends: 'airbnb-base'
 env:
-  node: false
-  browser: true
-  commonjs: true
+  node: true
+  browser: false
 rules:
 
+  # Forces alignment of objects values and colons of both single line objects
+  # and multiLine objects. This promotes smaller diffs.
+  key-spacing:
+    - 2
+    - singleLine:
+        beforeColon: false
+        afterColon: true
+      multiLine:
+        beforeColon: true
+        afterColon: true
+        align: colon
```

Comment:
I agree with you. Looking again at the default airbnb rules, the default is far better for small diffs. I deleted this custom ones.

I only left:
- `no-multiple-empty-lines` (default is 2 but I think 1 is better and more consistent).
- custom `comma-dangle` (only for node, as it gives syntax error as we don't transpile it, I fallback again to the default on browser files).